data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/configuration/S3ScanKeyPathOption.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.source.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+/**
+ * Class consists the scan include and exclude keys properties.
+ */
+public class S3ScanKeyPathOption {
+    @JsonProperty("include")
+    private List<String> s3scanIncludeOptions;
+    @JsonProperty("exclude")
```

Comment:
I believe that we discussed that this should be renamed to `exclude_suffix`.

Currently, `include` and `exclude` work very differently from each other. This would be confusing to a user. I'd expect them to have the same pattern matching approaches.